tests/acceptance/features/webUIFirstrunwizard/headingMessage.feature

```diff
@@ -0,0 +1,24 @@
+@webUI @insulated @disablePreviews
+Feature: default message from firstrunwizard is displayed after changing default popup message of firstrunwizard and reverting it back
+  As an administrator
+  I want to check if firstrunwizard displays default message on popup after changing the default message and reverting it
+  So that I can manipulate and send popup message from firstrunwizard to new users
+
+  Scenario: User logs into ownCloud for the first time
```

Comment:
This scenario is same as `tests/acceptance/features/webUIFirstrunwizard/firstrunwizard.feature:7`